guides/manual-installation.md

```diff
@@ -142,59 +152,115 @@ Restart postgres to apply the changes:
 systemctl reload postgresql
 ```
 
-### Install rabbitmq
+### Install RabbitMQ
 
 ```bash
 zypper install rabbitmq-server
 ```
 
-Enable and start the rabbitmq service:
+As the agent needs to reach RabbitMQ, allow connections from external hosts.
```

Comment:
>As the agent needs to reach RabbitMQ, allow connections from external hosts.
As the agent needs to reach RabbitMQ, connections from external hosts need to be allowed?
wdyt?